charts/cloudzero-agent/README.md

```diff
@@ -54,7 +63,7 @@ cloudAccountId: YOUR_CLOUD_ACCOUNT_ID
 clusterName: YOUR_CLUSTER_NAME
 # -- Region the cluster is running in.
 region: YOUR_CLOUD_REGION
-# -- CloudZero API key. Required if useExistingSecret is false.
+# -- CloudZero API key. Required if existingSecretName is false.
```

Comment:
Should this say `null` instead of `false`?

Reply:
yes, good catch, thanks!